packages/config-schema/src/schema/netlify-toml.schema.json

```
@@ -0,0 +1,646 @@
+          "type": "string",
+          "default": "netlify/functions"
+        }
+      },
+      "x-taplo": {
+        "links": {
+          "key": "https://docs.netlify.com/configure-builds/file-based-configuration/#functions"
+        }
+      }
+    },
+    "edge_functions": {
+      "type": "array",
+      "default": [{ "path": "/path", "function": "function-name" }],
+      "items": {
+        "title": "Edge Functions",
+        "description": "Edge Functions connect the Netlify platform and workflow with an open runtime standard at the network edge. This enables you to build fast, personalized web experiences with an ecosystem of development tools.",
+        "type": "object",
+        "properties": {
+          "path": {
+            "type": "string",
+            "description": "The path pattern that needs to match for this edge function."
+          },
+          "function": {
+            "type": "string",
+            "description": "The name of the function."
+          }
+        },
+        "additionalProperties": false,
+        "required": ["path", "function"]
+      },
+      "x-taplo": {
+        "links": {
+          "key": "https://docs.netlify.com/edge-functions/declarations/"
+        }
+      }
+    },
+    "dev": {
+      "title": "Netlify Dev",
+      "description": "Netlify Dev uses detectors to enable a local development environment for most tools and frameworks without any additional setup. You can use the `[dev]` section in `netlify.toml` for optional configuration.\n\nNote that `[dev]` doesn't run in the Bash shell, however, so you won't be able to use Bash-compatible syntax with the command.\n\nNetlify Dev also makes use of the functions directory setting to scaffold and serve your functions in a local development environment.",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "command": {
+          "title": "Dev command",
+          "description": "A command that starts your development server or runs a command such as a compiler watch in the background. If no `targetPort` is specified, it runs the command in the background in addition to the static file server.",
+          "type": "string"
+        },
+        "port": {
+          "title": "Port",
+          "description": "The port that Netlify Dev is accessible from in the browser.",
+          "type": "number",
+          "default": 8888
+        },
+        "targetPort": {
+          "title": "Target port",
+          "description": "port for your application server, framework, or site generator. If provided, the CLI will wait until the provided `targetPort` is reachable and then proxy requests to it. If you specify values for both `command` and `targetPort`, `framework` must be `#custom`.",
+          "type": "number"
+        },
+        "functionsPort": {
+          "title": "Functions port",
+          "description": "The port where Netlify Dev serves functions.",
```

Edit to be consistent

```suggestion
          "description": "Port where Netlify Dev serves functions.",
```